### ISSUE TITLE
macOS: Mouse click target off actual pointer in some screens

### ISSUE THREAD

**Issue by ashenomo** [OpenApoc/OpenApoc#569]
**Agent / vehicle assignment screen**
[Screen Recording 2019-04-28 at 22.46.42.mov.zip](https://github.com/OpenApoc/OpenApoc/files/3125046/Screen.Recording.2019-04-28.at.22.46.42.mov.zip)

* The white outline should be following the mouse cursor, but it seems to be off by some scaling factor
* Clicking selects the white outlined agent *and* the agent at which the cursor is actually pointing.

**Base view**
[Screen Recording 2019-04-28 at 22.56.06.mov.zip](https://github.com/OpenApoc/OpenApoc/files/3125052/Screen.Recording.2019-04-28.at.22.56.06.mov.zip)

* Base facility highlight is off by a constant factor?
* The Yes/No buttons of the "Destroy facility?" confirmation seem to be highlighted when the cursor is at the off location, but clicking here does not activate the button.
* Basically impossible to play 😞

**Comment by ashenomo** [comment]
Battle mode seems to work fine.

**Comment by ashenomo** [comment]
Original report was for windowed 1280x720. I also tried in Fullscreen mode and windowed 640x480 mode. Issues persisted in both cases.
